lua/cmake-tools/kits.lua

```diff
@@ -23,7 +23,9 @@ function kits.parse(global_kits_path, cwd)
   local config = nil
 
   local file = findcfg() -- check for config file
-
+  if file == nil then
+    return config
+  end
   file = file:gsub("~", vim.fn.expand("~"))
```

Comment:
Sorry, this bug is introduced here, you should move this line `file = file:gsub("~", vim.fn.expand("~"))` below `if file == nil then`